tests/php/Chat/ChatManagerTest.php

```diff
@@ -433,4 +434,125 @@ public function testClearHistory(): void {
 			);
 		$manager->clearHistory($chat, 'users', 'admin');
 	}
+
+	public function dataSearchIsPartOfConversationNameOrAtAll(): array {
+		return [
+			['a', 'test', true],
+			['h', 'test', true],
+			['A', 'test', false],
+			['H', 'test', false],
+			['T', 'test', true],
+			['t', 'test', true],
+			['notbeginingall', 'test', false],
+			['notbegininghere', 'test', false],
+			['notbeginingtest', 'test', false]
+		];
+	}
+
+	/**
+	 * @dataProvider dataSearchIsPartOfConversationNameOrAtAll
+	 */
+	public function testSearchIsPartOfConversationNameOrAtAll(string $search, string $roomDisplayName, bool $expected): void {
+		$reflector = new \ReflectionClass('OCA\Talk\Chat\ChatManager');
+		$method = $reflector->getMethod('searchIsPartOfConversationNameOrAtAll');
+		$method->setAccessible(true);
+
+		$actual = $method->invokeArgs($this->chatManager, [$search, $roomDisplayName]);
```

Comment:
There is a method in the base class for that:

```suggestion
		$actual = self::invokePrivate($this->chatManager, 'searchIsPartOfConversationNameOrAtAll', [$search, $roomDisplayName]);
```